test_repo/broken_module/model_view_odoo2.xml

```diff
@@ -100,51 +101,5 @@
                 </form>
             </field>
         </record>
-
-        <!-- duplicate record field "name" in *2M field-->
-        <record id="view_model_form5" model="ir.ui.view">
-            <field name="name">view.model.form</field>
```

Comment:
The valid cases are cases to test too

IMHO you shouldn't delete this part in order to double-confirm it is not detecting these kind of cases anymore